DecipherNow/gm-fabric-dashboard
# feat: lint commit msgs and automate changelog

Resolves #1333 

## Proposed Changes

  - added a changelog script that runs the cli tool conventional-changelog 
  - added a commit-msg husky hook to lint our commit messages
  
## How to Test

  - `npm run changelog` with the -u command to output an entry to the changelog with unreleased commits and check out CHANGELOG.md
  - try committing with an invalid commit message
